x/interchainqueries/keeper/keeper_test.go

```diff
@@ -82,9 +101,18 @@ func (suite *KeeperTestSuite) TestRegisterInterchainQuery() {
 
 		tt.malleate(contractAddress.String())
 
+		if tt.topupBalance {
+			senderAddress := suite.ChainA.SenderAccounts[0].SenderAccount.GetAddress()
+
+			coinsAmnt := sdktypes.NewCoins(sdktypes.NewCoin(sdktypes.DefaultBondDenom, sdktypes.NewInt(int64(10_000_000))))
+			bankKeeper := suite.GetNeutronZoneApp(suite.ChainA).BankKeeper
+
+			bankKeeper.SendCoins(ctx, senderAddress, contractAddress, coinsAmnt)
```

Comment:
could you please move this top up logic to a separate func? I can see four very same code blocks in this file making top ups